torch/csrc/cuda/Module.cpp

```diff
@@ -1,5 +1,3 @@
-#include <torch/csrc/python_headers.h>
```

Comment:
This file includes `cuda_runtime_api.h`. Changes in this file are just reordering & cleaning up of includes, in order to trigger clang-tidy for this file.